src/Tabs/Tabs.stories.mdx

```diff
@@ -3,13 +3,14 @@ import { withKnobs, boolean, select } from '@storybook/addon-knobs';
 import { Tabs } from './Tabs';
```

Comment:
Just keeping this one for another PR?

Reply:
Yeah, I think Tab is another quite complex case so thought it best to do it on its own